File: source/csharp/glympse/hal/GMutex.cs
//------------------------------------------------------------------------------
//
// Copyright (c) 2014 Glympse Inc.  All rights reserved.
//
//------------------------------------------------------------------------------

using System;
using System.Collections;
using System.Collections.Generic;
using System.Text;
using System.Threading;

namespace Glympse
{

/*J*public**/ interface GMutex : GCommon
{
     void block();
    
     void unblock();
};
    
/*C*typedef GMutex  GMutex;**/
    
}

File: source/java/com/glympse/java/hal/GMutex.java
//------------------------------------------------------------------------------
//
// Copyright (c) 2014 Glympse Inc.  All rights reserved.
//
//------------------------------------------------------------------------------

package com.glympse.java.hal;

import java.util.*;
import com.glympse.java.core.*;
import com.glympse.java.hal.*;

/*J*/public/**/ interface GMutex extends GCommon
{
    public void block();
    
    public void unblock();
};
    
/*C*typedef GMutex  GMutex;**/
    


